docs/plugins/discovery.md

```diff
@@ -0,0 +1,120 @@
+# Discovery
+Your App likely wants to integrate with the Platform's discovery capabilities. For example to add a "Watch Next" tile that links to your app from platform home screen.
+
+Getting access to this information requires to connect to lower level APIs made available by the platform. Since implementations differ between operators and platforms, the Lightning-SDK offers a Discovery plugin, that exposes a generic, agnostic interface to the developer.
+
+Under the hood, an underlaying transport layer will then take care of calling the right APIs for the actual platform implementation that your App is running on.
+
+The Discovery plugin is used to _send_ information to the Platform.
```

Comment:
How does this relate to the `Discovery.entitlements()` method _getting_ entitlements?

Reply:
The method only *sets* them. there's no way to get them.

Reply:
ah, I see, the docs on line 4 were wrong. I fixed them.